docs/source/gbd2017_models/causes/covid_19/index.rst

```diff
@@ -11,35 +11,285 @@ Covid-19 Custom SEIR Vivarium Cause Model
+
+#. Calculate deaths from infections and infection fatality ratio (IFR)
+
+Notably, the IHME forecast SEIR component includes a dual infectious state, :math:`I_{1}` and :math:`I_{2}`. The following equations represent the differential equations for each state in the SEIR component used in the IHME forecast model:
+
+			:math:`S` --> :math:`E` --> :math:`I_1` --> :math:`I_2` --> :math:`R`
+
+:math:`dS/dt = -(\beta(t)S(I_1+I_2)^\alpha)/N`
+
+:math:`dE/dt = (\beta(t)S(I_1+I_2)^\alpha)/N - \sigma E`
+
+:math:`dI_1/dt = \sigma E - \gamma_1 I_1`
+
+:math:`dI_2/dt = \gamma_1 I_1 - \gamma_2 I_2`
+
+:math:`dR/dt = \gamma_2 I_2`
+
+Where,
+
+.. list-table:: Parameter Definitions
+   :widths: 5 25
+   :header-rows: 1
+
+   * - Parameter
+     - Definition
+   * - :math:`S`
+     - Number of susceptible individuals in the population
+   * - :math:`E`
+     - Number of exposed individuals in the population
+   * - :math:`I_1`
+     - Number of infectious individuals in the first infectious state in the population 
+   * - :math:`I_2`
+     - Number of infectious individuals in the second infectious state in the population
+   * - :math:`R`
+     - Number of recovered individuals in the population
+   * - :math:`N`
+     - Number of individuals in all model states
+   * - :math:`\beta(t)`
+     - Degree of transmission (fit to/driven by model covariates). Approximately average number of contacts per person per time multiplied by the disease transmission probability of a contact between an infectious and susecptible individual
+   * - :math:`\alpha`
+     - Parameter to account for cluster nature of population transmission network. Ex: if alpha < 1, when the infectious population is large  the force of infection is attenuated
+   * - :math:`\sigma`
+     - Rate at which an exposed person becomes infectious (enters the :math:`I_1` state), i.e. :math:`1/\text{duration}_{E}`
+   * - :math:`\gamma_1`
+     - Rate at which an individual transitions from :math:`I_1` to :math:`I_2`, i.e. :math:`1/\text{duration}_{I_{1}}`
+   * - :math:`\gamma_2`
+     - Rate at which an infectious individual (:math:`I_2`) recovers, i.e. :math:`1/\text{duration}_{I_{2}}`
+
+.. todo::
+
+  Obtain precise definition of :math:`I_1` versus :math:`I_2`
+
+.. note::
+
+	In standard SEIR models, the parameter :math:`\beta(t)` defined here is often a *constant* parameter that represents the average number of contacts per person per time multiplied by the transmission probability of a contact between an infectious and susceptible individual (equal to :math:`R_0` when unit time is equal to the duration of the infectious period).
+
+	However, as described above, the :math:`\beta(t)` parameter is a function of time the considers the time-varying covariates used in the IHME forecasting model, which allows for the consideration of changes in relative mobility (which could be considered a proxy variable for the number of social contacts per person per time).
```

Comment:
For a given population/time, with a set mobility, temperature, etc. etc., does beta(t) yield a single value, or a distribution?

If we made beta(t) output a distribution, instead of the average value, given that this model is calculating cumulative deaths, would that make it any more accurate, or just more complex?

This then makes me want to ask if simulants could be initialized with the covariate values, with beta(t) calculated for each simulant... but again, given that we then aggregate the results, I don't have a good understanding of if that would make this more complicated & still estimate the same outcomes, on average.